hub/powertoys/grouppolicy.md

```diff
@@ -24,16 +28,27 @@ Since version 0.64, PowerToys is released on GitHub with [Administrative Templat
 3. Copy the "PowerToys.adml" file to the matching language folder in the PolicyDefinition folder. Create the folder if it doesn't already exist. (Example: %systemroot%\sysvol\domain\policies\PolicyDefinitions\EN-US)
 4. If your domain has more than one domain controller, the new ADMX files will be replicated to them at the next domain replication interval.
 
+### Import the administrative template in Intune
+
+You can find all instructions on how to import the tdministrative templates in Intune [here](https://learn.microsoft.com/mem/intune/configuration/administrative-templates-import-custom#add-the-admx-and-adml-files).
+
 ### Scope
 
 You will find the policies under "Administrative Templates/Microsoft PowerToys" in both the Computer Configuration and User Configuration folders. If both settings are configured, the setting in Computer Configuration takes precedence over the setting in User Configuration.
 
 ## Policies
 
+<!--Note for every dev who updates tis file:
+The syntax of OMA-URI is the following: ./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys[[~<category1>]~<categoryN>]/<ADMX-Displayname-ID>
+-->
+
 ### Configure global utility enabled state
 
-> [!NOTE]
```

Comment:
@alvinashcraft  should this be a notice or a subheader?

Reply:
This makes more sense as a note to me, unless you're going to discuss how things work in multiple different versions.

Reply:
I copied the layout from MS Edge group policy documentation.
I think if we have a notice for the version info on every policy this will make the docs page layout confusing. And the really important information aren't highlighted anymore. The version is important but not as important as the "PDF preview and Outlook incompatibility" for example.

Reply:
I would just make it an unformatted note/comment at the beginning or end of the policy's section then. Something like:

```
**Supported versions:** PowerToys 0.75.0 or later
```

Reply:
Note to me: What about removing only the header? We can (keep it as list item or make it italic or) use the comment style.

Reply:
The other option is to use different types of alert blocks to indicate the importance of the note. Use `[!NOTE]` for these and use `[!IMPORTANT]` for more urgent callouts. The downside of that approach is making it appear too busy.